src/tools/pub/cmd/index.md

```diff
@@ -1,34 +1,57 @@
 ---
 title: The pub tool
-description: Pub, a package management tool for Dart, supports a variety of commands.
+description: Pub, a package management tool for Dart, supports a variety of subcommands.
 ---
 
-The `pub` tool has commands for managing packages
+The pub tool has subcommands for managing packages
 and for deploying packages and command-line apps.
```

Comment:
I know you aren't writing this sentence here, but I couldn't help but feel it is a bit strange. Perhaps the double use of "packages", I'm not sure. 

Maybe just `The pub tool has subcommands for managing packages and command-line applications` would be fine, as I feel the term "managing" can also include deployment. 

You're the skillful copy editor though, so perhaps it's fine as is or you have a more clear suggestion :)

Reply:
Changing just one word didn't work for me. :) ptal

Reply:
Ugh, just noticed two "use"s in a row. Must... change...